lib/event.rb

```diff
@@ -0,0 +1,30 @@
+class Event
+
+  attr_reader :name, :crafts, :attendee_names
+
+  def initialize(name, craft, names)
+    @name = name
+    @crafts = [].push(craft).flatten
```

Comment:
Since we are passing an array of crafts in we can set `@crafts` to this array. Doing `[].push(craft).flateen` give us the same result but it more confusing to read. 

```suggestion
    @crafts = craft
```